.github/workflows/ci-sdk.yaml

```diff
@@ -0,0 +1,75 @@
+name: CI - Clarinet SDK
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "**/CHANGELOG.md"
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "**/CHANGELOG.md"
+
+  workflow_dispatch:
+
+jobs:
+  pre_run:
+    name: Cancel previous runs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.12.1
+        with:
+          access_token: ${{ github.token }}
+          persist-credentials: false
```

Comment:
This is an outdated way of enforcing concurrency. [You can swap it out for this method now.](https://github.com/hirosystems/clarinet/blob/main/.github/workflows/ci-clarinet-cli.yaml#L43-L45)